neard/src/config.rs

```diff
@@ -161,8 +149,19 @@ pub struct Network {
     pub external_address: String,
     /// Comma separated list of nodes to connect to.
     pub boot_nodes: String,
-    /// Maximum number of peers.
-    pub max_peers: u32,
+    /// Maximum number of active peers. Hard limit.
+    pub max_peer: u32,
```

Comment:
please add default serializer so that things don't break after this change.

Reply:
I was planning to rollback this particular rename! What do you mean by default serializer? Custom serializer?

Reply:
I guess it is deserializer to be more precise. Basically deserializer that adds a default value in case this field is not present in the input.